### PR TITLE
Readme and Changelog update

### PR DESCRIPTION
**Pull request by v-dareck**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## Windows/Linux 4.1.6 - 2017-02-03
+Updated PECL release packages. Here is the list of updates:
+### Added
+- Merge Linux and Windows code.
```
Comment (yitam) [inline CHANGELOG.md]: Use past tense for 'Merge' to be consistent with the rest

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,20 @@ All notable changes to this project will be documented in this file.
...
+- Enabled connection pooling with unixODBC. To enable pooling:
+  - in odbcinst.ini, add `Pooling=Yes` to the `[ODBC]` section and a positive `CPTimeout` value to `[ODBC Driver 13 for SQL Server]` section. See http://www.unixodbc.org/doc/conn_pool.html for detailed instructions.
+
+###Fixed
+- Fixed issues with sqlsrv_has_rows() to prevent it from moving statement cursor ([issue #37](https://github.com/Microsoft/msphpsql/issues/37)).
+- Fixed sqlsrv client buffer size to only allow positive integers ([issue #228](https://github.com/Microsoft/msphpsql/issues/228)).
+- Fixed PECL installation errors when PHP was installed from source ([issue #213](https://github.com/Microsoft/msphpsql/issues/213)).
+- Fixed segmentation fault with PDOStatement::getColumnMeta() when the supplied column index is out of range ([issue #224](https://github.com/Microsoft/msphpsql/issues/224)).
+- Fixed the assertion error (Linux) when fetching data from a binary column using the binary encoding ([issue #226](https://github.com/Microsoft/msphpsql/issues/226)).
+
```
Comment (yitam) [inline CHANGELOG.md]: Do I also need to add the issue numbers in PECL packages?

File: CHANGELOG.md (hunk 1 above)
Comment (yitam) [inline CHANGELOG.md]: One more thing, are we going to modify the version number in shared/version.h as well?

File: CHANGELOG.md (hunk 2 above)
Comment (v-dareck) [inline CHANGELOG.md]: No

**Pull request merged**